database/migrations/2020_06_03_182202_change_string_length_for_protection_seasons_column_on_plots_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeStringLengthForProtectionSeasonsColumnOnPlotsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('plots', function (Blueprint $table) {
+            $table->string('protection_seasons', 50)->nullable()->change();
```

Comment:
Let's leave it to default to 255 characters. And index it to filter or sort by it as needed in the future.
```suggestion
            $table->string('protection_seasons')->nullable()->change()->index();
```